RecoVertex/BeamSpotProducer/src/PVFitter.cc

```diff
@@ -87,6 +86,8 @@ void PVFitter::initialize(const edm::ParameterSet& iConfig,
   errorScale_        = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("errorScale");
   sigmaCut_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("nSigmaCut");
   fFitPerBunchCrossing=iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("FitPerBunchCrossing");
+  useOnlyFirstPV_    = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("useOnlyFirstPV");
+  minSumPt_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("minSumPt");
```

Comment:
Ciao @perrotta , 
A couple of answers/questions before making the fix:
- I agree sumPt needs to be corrected to a double
- I will clean out all not necessary commented lines
- hPVx and hPVy are not deleted
- I don't get your comment about untracked parameters. The point of this PR was to add 2 parameters (useOnlyFirstPV and minSumPt) to the PVFitter module in order to be able to cut on the PVs before the fit. I just added them the same way the other parameters (sigmaCut_ , errorScale_ ...) were already implemented.